CHANGES.md

```diff
@@ -8,6 +8,8 @@
 * Add `roundRobin` as a load balancer option.
 * Add the ability to route basted on the `dest` request header when using the
   TTwitter Thrift protocol
+* Change the default `dstPrefix` from the protocol name to `/s`
+* Change the default HTTP identifier to simply use the `Host` header
```

Comment:
Please indicate what identifier is used by default now. That was not easy for me to determine (i get it now, io.l5d.header?)